projects/storefrontlib/layout/breakpoint/breakpoint.service.ts

```diff
@@ -167,7 +167,7 @@ export class BreakpointService {
   }
 
   protected getMinSize(breakpoint: BREAKPOINT): number {
-    return (this.config[breakpoint] as BreakPoint)?.min;
+    return (this.config[breakpoint] as BreakPoint)?.min ?? 0;
```

Comment:
I don't think this should be returning 0. I think it should return null similarly to getMaxSize. Reason being in line 134 of this file, we sometimes do not want to return anything and if it is null, then it will be 0. However, if we set this 'default' value, then the behavior will change. It will no longer be 0, but 0 + 1.

Reply:
It should be `-1` because of this `this.getMinSize(prev) + 1 || 0`

Reply:
unresolving, because ` getSize(breakpoint: BREAKPOINT): number`  is now complaining for strict type on compilation. 

Reason being we made getMinSize potentially return null as well, so getSize need to be typed as `number | null`, which is fine because if you look at the test case in the spec file, it could have been falsy (which was returning null) https://github.com/SAP/spartacus/blob/develop/projects/storefrontlib/layout/breakpoint/breakpoint.service.spec.ts#L190. But need to handle another place for the breakpoint when we type getSize as `number | null` as well.